utils/getCoinGeckoTokenPrice.ts

```diff
@@ -3,17 +3,15 @@
 
 // so this can be tested outside browser
 import fetch from "isomorphic-fetch";
+const fetchOptions = {
+  // currently no options needed, previously we were setting content type and accept.
+};
 
 export async function getUmaPrice() {
   const query =
     "https://api.coingecko.com/api/v3/simple/price?ids=uma&vs_currencies=usd";
 
-  const response = await fetch(query, {
```

Comment:
Why remove these? Did they not work with these options? Or is this just not required anymore?

Reply:
While testing out this cors stuff, i removed the headers and did not notice any detrimental effects, It was working fine with them, but it just seemed unnecessary so I left them out.

Reply:
added this back in just in case